packages/cli/src/commands/account/authorize.ts

```diff
@@ -83,7 +83,8 @@ export default class Authorize extends BaseCommand {
         res.flags.blsPop
       )
     } else if (res.flags.role === 'validator') {
-      tx = await accounts.authorizeValidatorSigner(res.flags.signer, sig)
+      const validatorsWrapper = await this.kit.contracts.getValidators()
+      tx = await accounts.authorizeValidatorSigner(res.flags.signer, sig, validatorsWrapper)
```

Comment:
Passing in Validator New Required so that we dont need to have wrapperCache bring it in when constructing. this allows for the creation of MiniKit